yt/visualization/volume_rendering/shader_objects.py

```diff
@@ -1,9 +1,27 @@
+# encoding: utf-8
+"""
+Shader and ShaderProgram wrapper classes for vertex and fragment shaders used
+in Interactive Data Visualization
+"""
+
+# ----------------------------------------------------------------------------
+# Copyright (c) 2016, yt Development Team.
```

Comment:
this is a clear side-effect of mergers. Note that's there's another one like this in yt/visualization/volume_rendering/interactive_loop.py